### ISSUE TITLE
random seed

### ISSUE THREAD

**Issue by zzchuman** [ray-project/ray#15350]
Hello, I use the rllib to train my model and I want to repeat my experiment results. 
So, I set the random seed as:

torch.manual_seed(42)
np.random.seed(65)

 trainer = DQNTrainer(env="3x3grid", config={
        "framework": "torch",
        "seed": 42,
}

**Comment by rfali** [comment]
@zzchuman not an expert on this, but have you looked at this example (https://github.com/ray-project/ray/blob/master/rllib/examples/deterministic_training.py) and see if it gives deterministic results for you?
